app/routes/planet_routes.py

```diff
@@ -0,0 +1,93 @@
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request
+from app.routes.routes_helper import error_message
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
```

Comment:
Since this is the only blueprint in this file, consider naming the variable `bp`.